spec/rubocop/cop/style/commented_keyword_spec.rb

```diff
@@ -353,6 +352,36 @@ module Y
     RUBY
   end
 
+  it 'does not register an offense for RBS::Inline annotation for `end` keyword' do
+    expect_no_offenses(<<~RUBY)
+      def x
+        doubled_nums = [1, 2, 3].map do |num|
+          num * 2
+        end #: Array[Integer]
+      end
+    RUBY
+
+    expect_no_offenses(<<~RUBY)
+      def x
+      end #: String
+    RUBY
+
+    expect_no_offenses(<<~RUBY)
+      class X
+      end #: String
+    RUBY
+
+    expect_no_offenses(<<~RUBY)
+      module X
+      end #: String
+    RUBY
+
+    expect_no_offenses(<<~RUBY)
+      begin
+      end #: String
+    RUBY
```

Comment:
Can you write each added `it` and `expect_no_offenses` as a one-to-one pair? This helps ensure test independence and makes the purpose of each sample clearer.

Reply:
I think so. I fixed it according to your suggestion. 

And also I nested it blocks related to `RBS::Inline` in context, because test cases related to `RBS::Inline` has increased.

https://github.com/rubocop/rubocop/compare/e6d77c783a539ef06ad189327f84d89bdda7fb73..62ce67ba9b263d08d27fe96a0e85067977318627